### PR TITLE
Support different OS for the Mongo DB binary

### PR DESCRIPTION
**Pull request by FunkySayu**
Please test if it works for you @ArchangelX360 :)

### CODE REVIEW

**Review by ArchangelX360 — APPROVED** [review]

**Review by ArchangelX360 — COMMENTED** [review]
Untested (approved by mistake)

File: third_party/mongodb/BUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Rules used to manage a Mongo DB instance.
+licenses(["notice"])  # GNU AGPL v3.0
+
+# Since genrules does not support select in srcs, create an alias that
+# depends of the operating system that will be then used in the genrule.
+alias(
+    name = "server_files",
+    actual = select({
+        "//config:darwin": "@com_mongodb_darwin//:server_files",
```
Comment (ArchangelX360) [inline third_party/mongodb/BUILD]: @com_mongodb_darwin does not exist in WORKSPACE

**Review by ArchangelX360 — CHANGES_REQUESTED** [review]

File: config/BUILD [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+#
+config_setting(
+    name = "darwin",
+    values = {"cpu": "darwin"},
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "darwin_x86_64",
+    values = {"cpu": "darwin_x86_64"},
```
Comment (ArchangelX360) [inline config/BUILD:15]: I'm running a x64 OS but I am recognized as "darwin"

**Review by ArchangelX360 — APPROVED** [review]

File: config/BUILD (hunk 2 above)
Comment (FunkySayu) [reply config/BUILD:15]: Fixed in 7c8d476

**Pull request merged**